WalletWasabi.Fluent/Models/Transactions/PrivacySuggestionsModel.cs

```diff
@@ -62,30 +62,24 @@ public async Task<PrivacySuggestionsResult> BuildPrivacySuggestionsAsync(Transac
 
 		using (await _asyncLock.LockAsync(CancellationToken.None))
 		{
-			try
+			result.Add(VerifyLabels(info, transactionResult));
+			result.Add(VerifyPrivacyLevel(info, transactionResult));
+			result.Add(VerifyConsolidation(transactionResult));
+			result.Add(VerifyUnconfirmedInputs(transactionResult));
+			result.Add(VerifyCoinjoiningInputs(transactionResult));
+			foreach (var item in result)
 			{
-				result
-					.Combine(VerifyLabels(info, transactionResult))
-					.Combine(VerifyPrivacyLevel(info, transactionResult))
-					.Combine(VerifyConsolidation(transactionResult))
-					.Combine(VerifyUnconfirmedInputs(transactionResult))
-					.Combine(VerifyCoinjoiningInputs(transactionResult))
-					.Combine(VerifyChangeAsync(info, transactionResult, _linkedCancellationTokenSource));
+				yield return item;
 			}
-			catch (OperationCanceledException)
+			await foreach (var item in VerifyChangeAsync(info, transactionResult, _linkedCancellationTokenSource).ConfigureAwait(false))
 			{
-				Logger.LogTrace("Operation was cancelled.");
+				yield return item;
 			}
-			finally
```

Comment:
I removed this because it's irrelevant, the excpetion was never catched because it was always caught inside.
https://github.com/zkSNACKs/WalletWasabi/blob/33233a26c839aa06771847c6dd8099e61f7df45b/WalletWasabi/Blockchain/TransactionBuilding/ChangelessTransactionCoinSelector.cs#L91-L98